utils/port_test.go

```diff
@@ -13,3 +15,80 @@ func Test_GetFreePort(t *testing.T) {
 		t.Fatalf("Expected a port > 0 to be available, got %d", port)
 	}
 }
+
+func Test_FindPortInRange(t *testing.T) {
+	cases := []struct {
+		description string
+		s           string
+		port        int
+		errorMsg    string
+	}{
+		{
+			description: "single value",
+			s:           "6667",
+			port:        6667,
+			errorMsg:    "",
+		},
+		{
+			description: "csv",
+			s:           "6667,6668,6669",
+			port:        6667,
+			errorMsg:    "",
+		},
+		{
+			description: "range",
+			s:           "6668-6669",
+			port:        6668,
+			errorMsg:    "",
+		},
+		{
+			description: "invalid single",
+			s:           "abc",
+			port:        0,
+			errorMsg:    `strconv.Atoi: parsing "abc": invalid syntax`,
+		},
+		{
+			description: "invalid lower range",
+			s:           "abc-123",
+			port:        0,
+			errorMsg:    `strconv.Atoi: parsing "abc": invalid syntax`,
+		},
+		{
+			description: "invalid upper range",
+			s:           "123-abc",
+			port:        0,
+			errorMsg:    `strconv.Atoi: parsing "abc": invalid syntax`,
+		},
+		// Seems appvoyer's container lets you map to any port
```

Comment:
Probably a Windows thing. I think this can stay out of the tests for now as it's not portable. One thing to confirm manually is that we at least get a failure (as in, we're comfortable an `error` will be thrown if we try to allocate a port). If that's the case then i'm happy with this.

Reply:
Yeah I'm running on OSX and those ports should be reserved by system.  The test cases pass, will run again to verify